Library/Formula/slimerjs.rb

```diff
@@ -1,8 +1,23 @@
+class FirefoxDependency < Requirement
+  fatal true
+
+  def self.firefox_installation
+    paths = ["~/Applications/FirefoxNightly.app", "~/Applications/Firefox.app",
+             "/Applications/FirefoxNightly.app", "/Applications/Firefox.app"]
+    paths.find { |p| File.exist? File.expand_path(p) }
+  end
+
+  satisfy { firefox_installation }
```

Comment:
I tried ` self.firefox_installation ` a while ago without any luck, I'll try the fuller ` FirefoxDependency.firefox_installation ` shortly :+1: